### PR TITLE
CROWN-187 BE - Buy a resource page backend and database entity updates

### PR DESCRIPTION
**Pull request by fossterer**
- Updated models for ReceiverResource and ReceiverSupplier with new 'Person of Contact' fields
- Prefixed "poc_" to 'Person of Contact' specific properties except for "email" for backwards compatibility
- Updated setters and getters accordingly
- Added ToDos

### CODE REVIEW

File: .jhipster/ReceiverSupplier.json [hunk 1]
```diff
@@ -1,150 +1,210 @@
...
-    "jpaMetamodelFiltering": false,
-    "fluentMethods": true,
-    "readOnly": false,
-    "embedded": false,
-    "clientRootFolder": "",
-    "applications": "*"
+	"name": "ReceiverSupplier",
+	"fields": [
+		{
+			"fieldName": "orgName",
```
Comment (fossterer) [inline .jhipster/ReceiverSupplier.json:5]: To differentiate between say "name of org" and "name of person", I renamed the existing JSON keys too

File: src/main/java/org/crown/domain/ReceiverSupplier.java [hunk 2]
```diff
@@ -1,173 +1,208 @@
...
-    private String id;
+	@Field("poc_position")
+	private String pocPosition;
 
-    
-    @NotNull
-    @Field("name")
-    private String name;
+	@Field("poc_proof_of_association")
+	private String pocProofOfAssociation;
```
Comment (fossterer) [inline src/main/java/org/crown/domain/ReceiverSupplier.java:66]: https://crowndev.atlassian.net/browse/CROWN-187 - Person Of Contact Details - Item 2 sounds like a file upload field but I retained the "String" which was in use already.

Could it be that for this immediate MVP we don't spend time over this? We need to discuss on if we want to (1) store and retrieve Base64 content into/from database which can lead to limitations on size or (2) use Azure storage or such to actually upload the file and store/retrieve a pointer to it in database

File: src/main/java/org/crown/domain/ReceiverSupplier.java [hunk 3]
```diff
@@ -1,173 +1,208 @@
...
 
-    @Field("address")
-    private String address;
+	@Field("poc_proof_of_association_link")
+	private URI pocProofOfAssociationLink;
```
Comment (fossterer) [inline src/main/java/org/crown/domain/ReceiverSupplier.java:69]: @sharmmad This 'URI' is a trial as I haven't gotten my hands onto the database yet and there was no real validation. Can we have a discussion of how our database can be manually accessed in our breakout sessions this Sunday?

File: .jhipster/ReceiverSupplier.json (hunk 1 above)
Comment (fossterer) [reply .jhipster/ReceiverSupplier.json]: Removed specific prefix for "org" fields. Other comments still hold

**Comment by fossterer** [comment]
@sharmmad Ready for review

File: src/main/java/org/crown/domain/ReceiverSupplier.java (hunk 2 above)
Comment (sharmmad) [reply src/main/java/org/crown/domain/ReceiverSupplier.java:67]: Yes, very good point, we have story for this next sprint. We will discuss it in tomorrow's 1pm meeting

**Pull request merged**